tutorials/printing.adoc

```diff
@@ -1,83 +1,213 @@
+Ikona domečku představuje akci "home", která vyresetuje pozici tiskárny na `[0, 0, 0]` a zároveň sesynchronizuje fyzickou pozici tiskové hlavy s údaji v kontroléru tiskárny.
+
+Tlačítka `0.1`, `1`, `10` a `100` představují hodnotu o kterou posunete tiskovou hlavu následným kliknutím na šipku směru.
+
+V druhém sloupci lze ovládat extruder. 
+V číselném poli lze nastavit kolik milimetrů materiálu bude extrudováno/retractováno a tlačítky pod ním lze příslušnou akci provést.
+Ovládání extruderu je zvláště užitečné při vyjímání či zavádění filamentu.
+
+V posledním sloupci je užitečné tlačítko `Motors off`, který vypne krokové motory tiskárny a s tiskovou deskou a extruderem lze hýbat ručně.
+
+
+.Ovládání teplot v OctoPrint
+image::../images/octoprint-temps.png[Ovládací prvek teplot, 250, float="right"]
+==== Předehřátí tiskárny
+
+Předehřívání trvá v řádech minut. 
+Pro urychlení práce nebo pro zavedení či vyjmutí filamentu je nutné tiskárnu zahřát.
+
+Ve spodní části na záložce "Temperature" je možné nastavit teplotu, na kterou se má tiskárna zahřát.
+Vepsáním teploty do číselného pole ve sloupic "Target" a kliknutím na "Set" lze nastavit teplotu. 
+V rozbalovacím menu jsou pro rychlé nastavení přednastavené hodnoty pro ABS a PLA.
+
+
+==== Tisk
+
+Návod k tisku popisuje kapitola <<_tisk-krok-za-krokem>>. 
+Přečtěte si nyní zadání úkolů, abyste věděli co se od vás očekává, a pak pokračujte návodem jak tisknout krok za krokem.
+
+
+== Zadání úkolů
+
+Pro dnešní sadu úkolů se vtělte do pracovitého zaměstnance QC oddělení fiktivní firmy "Josef Prča Research", kontrolující výstupní kvalitu 3D tiskáren.
+Vaším úkolem bude ověřit, zda tiskárna tiskne správně a dále ověřit, jak velký převis bez podpor vaše tiskárna zvládne vytisknout.
+
+TODO: Trapnej vtípek na průšu asi vynecháme.
+
+
+=== Úkol 1 (1 bod)
+
+*Potřebujete ověřit, zda tiskárna skutečně vytiskne stejné rozměry, které jsou v modelu.*
+
+Jako testovací objekt jste si vybrali link:../stls/printing/20mm-box.stl[model kostky o přesných rozměrech 20×20×10 mm].
+K tiskárně jste dostali tiskový profil, který na ní byl doteď používán. 
+Předpokládejme, že není potřeba ho upravovat po kalibrační stránce (hodnoty jako velikost trysky a údaje o materiálu jsou správné). 
+
+Abyste ušetřili materiál, nastavte tisk tak, aby se vytiskly kostky 4 a každá z nich byla dutá, měla jednu obvodovou vrstvu a žádnou vrchní vrstvu (ve výsledku byste měli dostat 4 hranaté kalíšky).
+
+
+=== Úkol 2 (2 body)
+
+*Dále chcete ověřit, jaký největší převis bez využití podpor tiskárna zvládne vytisknout.*
+
+Vytiskněte model z portálu Thingiverse -- https://www.thingiverse.com/thing:1564848[Overhang Angle Test Object] (link:../stls/printing/Overhang_Angle_Test_Object.zip[mirror zde]) bez podpor. 
+Zjistěte jaký největší úhel převisu je tiskárna schopna vytisknout.
+
+Po vytištění přidejte do modelu podpory, ale pouze na místech, kde je to potřeba. Upravená tisková data vytiskněte.
```

Comment:
Valíme podpory teda už na prvním tiskovém cviku?

Reply:
Já nevím, respektoval jsem, co bylo napsaný v dokumentu.

Reply:
tohle tam bude, ale v úkolu na podpory. tedy v současnosti je to na předělání tutorials/supports